actors/builtin/multisig/multisig_test.go

```diff
@@ -723,6 +767,45 @@ func TestRemoveSigner(t *testing.T) {
 			expectApprovals: int64(2),
 			code:            exitcode.ErrNotFound,
 		},
+		{
+			desc: "remove signer and decrease threshold to 0",
+
+			initialSigners:   []addr.Address{anne, bob, chuck},
+			initialApprovals: int64(1),
+
+			removeSigner: chuck,
+			decrease:     true,
+
+			expectSigners:   []addr.Address{anne, bob},
+			expectApprovals: int64(0),
```

Comment:
seems not ok

Reply:
Good catch, `decrease=true` should be rejected when the approvals is `1`.

Reply:
Or, for consistency with `increase`, `decrease` should be ignored when it would lead to an illegal state. See #114